### DISCUSSION TITLE
How to add authentication to the web UI ?

### DISCUSSION THREAD

**Discussion by Fx4-X** [mindsdb/mindsdb#1399] [Get help] [answered]
Hello,

How to add authentication to the web UI ?

**Comment by ZoranPandovski** [comment, marked as answer]
Hello @Fx4-X sorry for the late response. Atm, this is not possible with the current version but is on our to-do. In the meantime, if you like you can use a gateway to enable this(for e.g Kong). If you have any other questions or you need help with setting up we can jump on a call to help you :raised_hands: .

**Reply by Fx4-X** [reply]
@ZoranPandovski Thanks for the feedback.

I managed to set up http basic auth using nginx, it was easy for me as MindsDB Scout (GUI) was already behind the nginx reverse proxy.
if anybody came across similar scenario here is a nice guide from nginx https://docs.nginx.com/nginx/admin-guide/security-controls/configuring-http-basic-authentication/
